neqo-transport/src/cc/mod.rs

```diff
@@ -0,0 +1,67 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// Congestion control
+#![deny(clippy::pedantic)]
+
+use crate::path::PATH_MTU_V6;
+use crate::tracking::SentPacket;
+use neqo_common::qlog::NeqoQlog;
+
+use std::fmt::{Debug, Display};
+use std::time::{Duration, Instant};
+
+mod classic_cc;
+mod new_reno;
+
+pub use classic_cc::ClassicCongestionControl;
+pub use classic_cc::{CWND_INITIAL_PKTS, CWND_MIN};
+
+pub const MAX_DATAGRAM_SIZE: usize = PATH_MTU_V6;
+
+pub trait CongestionControl: Display + Debug {
+    fn set_qlog(&mut self, qlog: NeqoQlog);
+
+    fn cwnd(&self) -> usize;
+
+    #[cfg(test)]
+    fn ssthresh(&self) -> usize;
```

Comment:
Is this something we can leave on the specific implementation?  That is, this is an internal detail that we might want to look at, but do we need to have it generically?

Reply:
This was used by some tests. This should not be exposed. I will remove it.